csv/generic/shared/src/test/scala/fs2/data/csv/generic/CsvRowDecoderTest.scala

```diff
@@ -18,6 +18,7 @@ package fs2
 package data.csv
```

Comment:
I would still like to see these test cases moved out of the `generic` module please, they just don't belong here.

Reply:
Done! We used the decoder derive methods, but now I've added a custom decoder, as this method is no longer available in `csvJVM`. Please let me know if this is also what you had in mind.